cookbook/external-media-storage.rst

```diff
@@ -0,0 +1,44 @@
+Store Media in an external Storage
+==================================
+
+Sulu is able to upload newly created media files directly to external storage provider (such as google cloud-storage).
+
+Currently available storages:
+
+* Google Cloud-Storage
+
+Google Cloud-Storage
+--------------------
+
+First follow this google-cloud documentation https://cloud.google.com/video-intelligence/docs/common/auth#set_up_a_service_account
+to setup a system-account and download a json-key.
+
+This file should look like this:
```

Comment:
Not clear what file you are talking about - I guess it relates to the json-key above?